### PR TITLE
dev-libs/argtable: fix call to undeclared library function strcmp

### PR DESCRIPTION
**Pull request by listout**
Closes: https://bugs.gentoo.org/885609

### CODE REVIEW

File: dev-libs/argtable/files/argtable-2.13-Fix-undeclared-library-function.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
+From 519609d844f1e5bbf37407de8e43fa2d2be03262 Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline dev-libs/argtable/files/argtable-2.13-Fix-undeclared-library-function.patch:1]: Have you sent this upstream? If not, why not? If so, include a link :)

Comment (listout) [reply dev-libs/argtable/files/argtable-2.13-Fix-undeclared-library-function.patch:1]: No. I'm confused about the upstream. Is it https://github.com/argtable/argtable3 or https://github.com/jonathanmarvens/argtable2. If it's the latter, then upstream seems dead.

Comment (thesamesam) [reply dev-libs/argtable/files/argtable-2.13-Fix-undeclared-library-function.patch:1]: The homepage and src uri are sourceforge?

Comment (listout) [reply dev-libs/argtable/files/argtable-2.13-Fix-undeclared-library-function.patch:1]: Yes. The last release was January 10th, 2011.

Comment (thesamesam) [reply dev-libs/argtable/files/argtable-2.13-Fix-undeclared-library-function.patch:1]: so why are we talking about github?

**Pull request closed without merging**